CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.21.5] - 2021-08-31
+- ConstantQpsDarkClusterStrategy post-prod fixes
+  - DarkClusterConfig's dispatcherOutboundTargetRate: int -> float
```

Comment:
Let's make the changelog more obvious. e.g. Change the type of dispatcherOutboundTargetRate in DarkClusterConfig.pdl from int to float.

Reply:
Changelog updated verbatim